integration-tests/tests/register-user.js

```diff
@@ -23,21 +28,36 @@ const testRegisterUserFn = withUser(async ({ page, testUrl, token }) => {
   // We wait here for the cookie message to disappear, it has a fixed duration fading animation of 300ms
   await delay(350);
   await verifyAccessibility(page);
-  await click(page, clickable({ textContains: 'View Workspaces' }));
+
+  // This is the hamburger menu
+  await click(page, ariaLabel({ labelContains: 'Toggle main menu' }));
```

Comment:
I would recommend moving the label to the button so that you can use normal clickable methods. I verified that it still works just fine with the screen reader, and axe reports no issues.

![image](https://github.com/DataBiosphere/terra-ui/assets/484484/ed6e4552-f946-4f74-9cf8-8b18438d6b26)

Reply:
Just tried it, and register-user passes!